client/components/ChallengesList.js

```diff
@@ -85,47 +103,66 @@ const component = ({
         <h1>Our Challenges</h1>
       </div>
       <Search history={history} searchTerm={searchTerm} />
-      <div className="d-flex justify-content-around">
+      <div>
         {challenges.map(challenge => {
           const imageSrc = challenge.images[0].data;
           return (
-            <div className="card" style={{ width: '20rem' }} key={challenge.id}>
+            <div
+              className="card d-inline-flex"
+              style={{ width: '25%', marginRight: '1em', marginBottom: '1em' }}
+              key={challenge.id}
+            >
               <div className="card-body">
                 <img src={imageSrc} alt="" className="card-image-top" />
                 <h5 className="card-title">{challenge.name}</h5>
                 <p className="card-text">{challenge.description}</p>
                 <Link to={`/challenges/${challenge.id}`} className="btn btn-primary">
                   Go to Challenge
                 </Link>
-                <div>
-                  {solutionByChallengeId(challenge.id) ? (
-                    <div>
-                      <p>Statistic</p>
-                      <p>
-                        Attempted: {attemptedTimes(solutionByChallengeId(challenge.id))}{' '}
-                        {attemptedTimes(solutionByChallengeId(challenge.id)) > 1 ? 'times' : 'time'}
-                      </p>
-                      <p>
-                        Attempted by number of Users:{' '}
-                        {attemptedByUsers(solutionByChallengeId(challenge.id))}
-                      </p>
-                      <p>Average Score: {avgScore(solutionByChallengeId(challenge.id))}</p>
+                <button type="button" onClick={() => collapseController(challenge.id)}>
+                  More info
+                </button>
+                <Collapse
+                  isOpened={collapseStatus[challenge.id] ? collapseStatus[challenge.id] : false}
+                >
+                  <div>
+                    {solutionByChallengeId(challenge.id) ? (
                       <div>
-                        {solutionCompleted(solutionByChallengeId(challenge.id), user.id) ? (
-                          <MdCheckmarkCircle fontSize="30px" color="#43853d" />
-                        ) : null}
+                        <p>Statistic</p>
+                        <p>
+                          Attempted: {attemptedTimes(solutionByChallengeId(challenge.id))}{' '}
+                          {attemptedTimes(solutionByChallengeId(challenge.id)) > 1
+                            ? 'times'
+                            : 'time'}
+                        </p>
+                        <p>
+                          Attempted by number of Users:{' '}
+                          {attemptedByUsers(solutionByChallengeId(challenge.id))}
+                        </p>
+                        <p>Average Score: {avgScore(solutionByChallengeId(challenge.id))}</p>
+                        <div>
```

Comment:
The check mark should be visible when this section is collapsed, so should move above `<Collapse />`